examples/explanation.py

```diff
@@ -0,0 +1,75 @@
+import os.path as osp
+
+import matplotlib.pyplot as plt
+import torch
+import torch.nn.functional as F
+
+from torch_geometric.datasets import Planetoid
+from torch_geometric.explain import Explainer, GNNExplainer
+from torch_geometric.nn import GCNConv
+
+
+# print some dataset statistics
+def data_stats(dataset):
+    print(f"Number of features: {dataset.num_features}")
+    print(f"Number of classes: {dataset.num_classes}")
+    print(f"Number of instances in dataset: {dataset.len}")
+    print(f"Single sample #0: {dataset[0]}")
+    print(f"Number of nodes in single sample: {dataset[0].num_nodes}")
+    print(f"Number of edges in single sample: {dataset[0].num_edges}")
+
+
+dataset = 'Cora'
+path = osp.join(osp.dirname(osp.realpath(__file__)), '..', 'data', 'Planetoid')
+dataset = Planetoid(path, dataset)
+data_stats(dataset)
+data = dataset[0]
+
+
+class Net(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.conv1 = GCNConv(dataset.num_features, 16)
+        self.conv2 = GCNConv(16, dataset.num_classes)
+
+    def forward(self, x, edge_index, edge_weight):
+        x = F.relu(self.conv1(x, edge_index, edge_weight))
+        x = F.dropout(x, training=self.training)
+        x = self.conv2(x, edge_index, edge_weight)
+        return F.log_softmax(x, dim=1)
+
+
+device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+model = Net().to(device)
+data = data.to(device)
+optimizer = torch.optim.Adam(model.parameters(), lr=0.01, weight_decay=5e-4)
+
+for epoch in range(1, 201):
+    model.train()
+    optimizer.zero_grad()
+    log_logits = model(data.x, data.edge_index, data.edge_weight)
+    loss = F.nll_loss(log_logits[data.train_mask], data.y[data.train_mask])
+    loss.backward()
+    optimizer.step()
+
+explainer = Explainer(
+    model=model,
+    algorithm=GNNExplainer(epochs=200),
+    explanation_type='model',
+    node_mask_type='attributes',
+    edge_mask_type='object',
+    model_config=dict(
+        mode='multiclass_classification',
+        task_level='node',
+        return_type='log_probs',
+    ),
+)
+
+# which node's neightborhood to visualize
+node_index = 71
+explanation = explainer(data.x, data.edge_index, index=node_index,
+                        edge_weight=data.edge_weight)
+print(f'Generated explanations in {explanation.available_explanations}')
+
+explanation.visualize_subgraph(node_index, data.edge_index)
+print('Visualizing subgraph')
```

Comment:
Can we just add this to `examples/gnn_explainer.py` instead of creating a new example?